@navikt/core/react/src/layout/box/Box.tsx

```diff
@@ -122,15 +122,15 @@ export const Box: OverridableComponent<BoxProps, HTMLDivElement> = forwardRef(
     const boxBackground =
       typeof background === "string" ? `var(--a-${background})` : undefined;
     const boxBorderColor =
-      typeof borderColor === "string"
-        ? `var(--a-${borderColor})`
-        : "transparent";
+      typeof borderColor === "string" ? `var(--a-${borderColor})` : undefined;
+    const borderStyle = typeof borderColor === "string" ? "solid" : undefined;
```

Comment:
Kan hende dette vil kunne skape litt problemer i fremtiden hvis vi legger til en egen prop for borderWidth 🤔 Da siden border-width blir styrt av borderColor og man da må ha en codemod som setter `borderWidth="1"` for alle som oppdaterer.

Vil en egen prop `borderWidth?: "1"` som er default `undefined` fungere? Kan da utvide den i etterkant hvis vi ønsker flere border-bredder (f.eks ErrorSummary)